src/utils/assetIcons.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from 'vue';
+import type { Dictionary, Protocol } from '@/types';
+import { PROTOCOLS } from '@/constants';
+
+import AeternityIcon from '@/icons/coin/aeternity.svg?vue-component';
+import BitcoinIcon from '@/icons/coin/bitcoin.svg?vue-component';
+import EthereumIcon from '@/icons/coin/ethereum.svg?vue-component';
+import LexonTokenIcon from '@/icons/tokens/ct_xtk8rSz9suPb6D6VLquyfVji25FcnFRDjn3dnn5mmvHsPiESt.svg?vue-component';
+
+const coinIcons: Record<Protocol, Component> = {
+  [PROTOCOLS.aeternity]: AeternityIcon,
+  [PROTOCOLS.bitcoin]: BitcoinIcon,
+  [PROTOCOLS.ethereum]: EthereumIcon,
+};
+
+const assetIcons: Dictionary<Component> = {
+  ...coinIcons,
+  ct_xtk8rSz9suPb6D6VLquyfVji25FcnFRDjn3dnn5mmvHsPiESt: LexonTokenIcon,
+};
+
+export function getAssetIcon(contractId: string): Component {
+  return assetIcons[contractId];
+}
+
+export function hasAssetIcon(contractId: string): boolean {
+  return !!assetIcons[contractId];
+}
```

Comment:
If this file is not a class/lib/composable I don't see much benefit of having getter utils. I mean I don't like the fact that this file acts both as a source of constants and functions. I see following options:
1. Split this file by moving the `coinIcons` & `assetIcons` to constants file and move the functions to `utils/common`.
2. Leaving only the constants and moving them to `constants` file. Accessing the values would be simple as just writing `ASSET_ICONS[contractId]`.

If you choose to leave the methods please use `AssectContractId` for the `contractId` arguments.

Reply:
It doesn't export any constants, only the functions

Reply:
Yeah, but it has a constants that looks like an app constants that we store in the constants/ dir.